test/scripts/install-kind.sh

```diff
@@ -15,5 +15,5 @@ popd
 
 go get sigs.k8s.io/kind@v0.9.0
 pushd ../contrib
-./kind.sh
+./kind.sh -ho
```

Comment:
we dont always want to enable hybrid overlay right? Just for e2e control plane tests which should be handled via ENV var?

Reply:
oops, yeah, that's something I put for testing locally